settings.yml

```diff
@@ -0,0 +1,23 @@
+project: lib_device_control
+version: 5.0.0
+
+## example of overridable setting
+# watermarks is redefined in documentation and again for a specific product
+# eg different products might have different setting but we'd like to provide defaults
+watermarks: []
+
+products:

```

Comment:
I would not put a product, as it is a library, it does not need different document generations. use lib_name directly. 
check:
https://github.com/xmos/lib_awe/blob/develop/settings.yml